src/app/sign_up/sign_up.html

```diff
@@ -0,0 +1,36 @@
+<section class="sign_up">
+  <h1 class="sign-up-header">sign up</h1>
+  <div class="form-container">
+    <form name="signUpForm" ng-submit="signUpForm.$valid && signUpCtrl.createUser()" novalidate>
+      <label for="name" class="form-label" ng-hide="(signUpForm.$submitted || signUpForm.name.$touched) && signUpForm.name.$error.required">name</label>
+      <label for="name" class="form-label error" ng-show="(signUpForm.$submitted || signUpForm.name.$touched) && signUpForm.name.$error.required">name is required</label>
```

Comment:
Since the error message it's not a really a label for the input (in the way that describes the input) it's better to display error messages in, for example, a span